hnn_core/neuron.py

```diff
@@ -317,6 +316,39 @@ def __exit__(self, cell_type, value, traceback):
         if _LAST_NETWORK is not None:
             _LAST_NETWORK._clear_neuron_objects()
 
+    def set_cell_morphology(self, cell_morphology=None):
```

Comment:
in the future, we can expand this to have modeldb files, but we are not there yet ...

Reply:
Is there a particular reason why this is a method of `NeuronNetwork` instead of the lower-level `Network` class?

Reply:
`NeuronNetwork` class is the one which deals with Neuron imports, no? [Down below](https://github.com/jonescompneurolab/hnn-core/blob/master/hnn_core/neuron.py#L372-L374) in this same class is where it was previously instantiated.